pwndbg/commands/slab.py

```diff
@@ -292,3 +299,35 @@ def slab_list(filter_) -> None:
         )
 
     print(tabulate(results, headers=["Name", "# Objects", "Size", "Obj Size", "# inuse", "order"]))
+
+
+def slab_contains(address: str) -> None:
+    """prints the slab_cache associated with the provided address"""
+    try:
+        addr = int(gdb.parse_and_eval(address))
+    except gdb.error:
+        print(M.error(f"Could not parse '{address}'"))
+        return
+
+    min_pfn = 0
+    max_pfn = int(gdb.lookup_global_symbol("max_pfn").value())
+
+    start_addr = pwndbg.gdblib.kernel.pfn_to_virt(min_pfn)
+    end_addr = pwndbg.gdblib.kernel.pfn_to_virt(
+        max_pfn + pwndbg.gdblib.kernel.arch_ops().page_size()
+    )
+
+    if not start_addr <= addr <= end_addr:
```

Comment:
Btw shouldn't this maybe be `start_addr<=addr<end_addr` :P?

Reply:
you're absolutely right